bin/clean.js

```diff
@@ -0,0 +1,12 @@
+var rimraf = require('rimraf');
+var fs = require('fs');
+
+var dir = '../build';
+
+rimraf(dir, function(err){
+	if(err)
```

Comment:
Curly braces around the body.
Here is my codestyle https://github.com/nicksp/fe-codestyle/blob/master/javascript.md